haas/cli.py

```diff
@@ -124,6 +117,8 @@ def serve_networks():
     """Start the HaaS networking server"""
     from haas import model, deferred
     from time import sleep
+    server.register_drivers()
```

Comment:
Should we just call server.api_server_init() and add a stop_consoles=True default which gets overridden here?